lms/djangoapps/certificates/docs/decisions/004-cert-status.rst

```diff
@@ -0,0 +1,56 @@
+Course certificate Status
+=========================
+
+Status
+------
+Accepted
+
+Background
+----------
+Course certificates can have any one of a number of status values (for the
+full list, see the `CertificateStatuses model`_). For a certificate to be
+available for the user to see, it must be in the *downloadable* state, meaning
+it must have a status value of *downloadable*.
+
+Decision
+--------
+The course certificate code will only set the following certificate statuses:
+
+* downloadable - The user has been granted this certificate and the certificate is ready and available
+* notpassing - The user has not achieved a passing grade
+* unavailable - Certificate has been invalidated
+* unverified - The user does not have an approved, unexpired identity verification
+
+Other status values are kept for historical reasons and because existing course
+certificates may have been granted that status.
+
+Consequences
+------------
+If certificate has been invalidated, its status will be *unavailable*.
```

Comment:
Nit: Are we missing an `a` at the beginning of the sentence?

Reply:
Yep, will fix